integr8ly/integreatly-operator
# [MGDAPI-3787] include omitted unit tests

# Issue link
https://issues.redhat.com/browse/MGDAPI-3787

# What
Unit test that are created outside of the `pkg` package are never called. Examples of these unit test can be found in the `controllers` package. The unit test that are outside of the `pkg` package also are not included in the coverage.

# Verification steps
- Run `make test/unit`
- Ensure the unit tests in packages `apis`, `controllers` and `pkg` are ran with coverage enabled
